README.md

```diff
@@ -4,7 +4,16 @@ This is a Nextflow pipeline for generating sequencing reports for the SNP&amp;Se
 ## Pre-requisites
 You need to:
   - install Nextflow (e.g. using conda `conda create -n nextflow-env nextflow` or downloading from [nextflow.io](https://www.nextflow.io/)).
+  - Make Nextflow binary executable:
+  ```bash
+    chmod +x /usr/local/bin/nextflow
+  ```
   - install [Singularity (version > 2.6)](https://singularity.lbl.gov/install-linux#adding-the-mirror-and-installing).
+  - Install test requirements:
+  ```bash
+    pip install -r requirements-dev.txt
```

Comment:
This is not required for a normal user, only developers. I saw now that this is command is part of the `Running tests locally` section. I think this can be removed.